clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -2361,6 +2386,19 @@ void CIRGenModule::setFunctionAttributes(GlobalDecl GD, mlir::cir::FuncOp F,
                                          bool IsIncompleteFunction,
                                          bool IsThunk) {
   assert(!MissingFeatures::setFunctionAttributes());
+  auto D = GD.getDecl();
+  const VisibilityAttr *VA = D->getAttr<VisibilityAttr>();
+  CIR_VisibilityAttr cir_visibility;
```

Comment:
We use camelBack: https://llvm.github.io/clangir/GettingStarted/coding-guideline.html#style, this should be `cirVisibility` instead.